app/javascript/schema.json

```diff
@@ -36,51 +36,6 @@
           "name": "Query",
           "description": null,
           "fields": [
-            {
-              "name": "announcements",
```

Comment:
This is a side effect of changing the schema so I feel like it should be part of the schema change commit.